src/components/dialogs/substation/modification/substation-modification-form.js

```diff
@@ -74,8 +75,8 @@ const SubstationModificationForm = ({
                 );
                 canRemoveLine = false;
             } else {
-                // we can mark as deleted only real prop (having a previous value)
-                if (properties[idx][PREVIOUS_VALUE]) {
+                // we can mark as deleted only prop having a previous value, not added in current modification
+                if (properties[idx][PREVIOUS_VALUE] && properties[idx][ADDED] === false) {
```

Comment:
```suggestion
                if (
                    properties[idx][PREVIOUS_VALUE] &&
                    properties[idx][ADDED] === false
                ) {
```

Reply:
yes the CI is KO because of this, I missed it